### DISCUSSION TITLE
Welcome to derpyaxolotl.github.io Discussions!

### DISCUSSION THREAD

**Discussion by DerpyAxolotl** [DerpyAxolotl/derpyaxolotl.github.io#2] [Announcements]
Justin is balding.  We need to buy him a nice wig to help him cope.

**Comment by AMagnitude67Earthquake** [comment]
> > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > Justin is balding. We need to buy him a nice wig to help him cope.
you have been quoted.

**Reply by AMagnitude67Earthquake** [reply]
> that did not work how i intended it to. but it seems fine.
